code/modules/projectiles/guns/projectile/shotgun.dm

```diff
@@ -84,7 +85,7 @@
 
 	if(chambered)//We have a shell in the chamber
 		chambered.forceMove(get_turf(src)) //Eject casing
-		playsound(src.loc, chambered.drop_sound, DROP_SOUND_VOLUME, FALSE, required_asfx_toggles = ASFX_DROPSOUND)
+		playsound(src.loc, casing_drop_sound, DROP_SOUND_VOLUME, FALSE, required_asfx_toggles = ASFX_DROPSOUND)
```

Comment:
Same as above

Reply:
When tested, shells from boxes played the sound fine - when pump-actioned out of a shotgun they played the incorrect (metallic) sound. This here fixed that, at least for me